docs/using/recipes.md

```diff
@@ -528,3 +528,18 @@ USER $NB_UID
 
 RUN pip install --quiet --no-cache-dir auto-sklearn
 ```
+
+## Enable Delta Lake in Spark notebooks
+Please note that the Delta lake packages are only available for Spark version > 3.0. By adding the properties to `spark-defaults.conf`, the user no longer needs to enable Delta support in each notebook. 
+
+```dockerfile
+FROM jupyter/pyspark-notebook:latest
+
+USER root
+
+RUN echo 'spark.jars.packages io.delta:delta-core_2.12:0.8.0' >> $SPARK_HOME/conf/spark-defaults.conf && \
```

Comment:
Manage the version as an `ARG` is a good practice.

```suggestion
ARG delta-core_version="0.8.0"

USER root

RUN echo "spark.jars.packages io.delta:delta-core_2.12:${delta-core_version}" >> $SPARK_HOME/conf/spark-defaults.conf && \
```

Reply:
The Scala version will bump over time as well:

```
ARG scala_version="2.12"
ARG delta-core_version="0.8.0"
USER root
RUN echo "spark.jars.packages io.delta:delta-core_${scala_version}:${delta-core_version}" >> $SPARK_HOME/conf/spark-defaults.conf && \ 
```